common/djangoapps/student/admin.py

```diff
@@ -2,6 +2,9 @@
 from django import forms
 from django.contrib.auth import get_user_model
 from django.contrib.auth.admin import UserAdmin as BaseUserAdmin
+from django.contrib.auth.forms import UserCreationForm as BaseUserCreationForm
```

Comment:
you can do imports inline for example
`from django.contrib.auth.forms import UserCreationForm as BaseUserCreationForm, UserChangeForm as BaseUserChangeForm`

Reply:
ok, done